pkg/controller/command/vdri/models.go

```diff
@@ -25,7 +24,7 @@ type CreatePublicDIDArgs struct {
 // CreatePublicDIDResponse for returning public DID created
 type CreatePublicDIDResponse struct {
 	// TODO return base64-encoded raw bytes of the DID doc [Issue: #855]
-	DID *did.Doc `json:"did"`
+	DID json.RawMessage `json:"did"`
```

Comment:
Is this TODO still relevant?
`// TODO return base64-encoded raw bytes of the DID doc [Issue: #855]`

Reply:
you right its not valid any more so i removed comment and put issue number to close